src/Refitter.Core/SchemaCleaner.cs

```diff
@@ -0,0 +1,172 @@
+﻿using NJsonSchema;
+
+using NSwag;
+
+namespace Refitter.Core;
+
+public class SchemaCleaner
+{
+    private readonly OpenApiDocument _doc;
```

Comment:
@kirides this is just me nit-picking, but can we rename `_doc` to `document`?

Reply:
done